src/app/vault/add-edit-custom-fields.component.html

```diff
@@ -85,6 +85,7 @@
             appBlurClick
             role="button"
             appA11yTitle="{{ 'toggleVisibility' | i18n }}"
+            [attr.aria-pressed]="showValue"
```

Comment:
PR is looking great. Just one minor thing, I discovered during testing. The button would not announce it's state.

```suggestion
            [attr.aria-pressed]="f.showValue"
```

Reply:
ah, good catch. corrected and tested, now works as it should